_includes/scenarios/integrate/propose-attribute-to-peer.md

```diff
@@ -1,5 +1,215 @@
+{: .notice--info}
+
+<!--- TODO: Insert Links to "Read Attribute from peer" and "Create Attribute for peer" guide --->
+
+### Reject a ProposeAttributeRequestItem
+
+Even if the Recipient accepts the Request for proposing Attributes as a whole, it may decide not to deal with all of the Attributes proposed by the Sender. To be more precise, the Recipient has the option of rejecting [ProposeAttributeRequestItems]({% link _docs_integrate/data-model-overview.md %}#proposeattributerequestitem) that have the value `false` specified in their `mustBeAccepted` property. To reject a ProposeAttributeRequestItem, use the [RejectRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#rejectrequestitemparameters). The rejection of a ProposeAttributeRequestItem leads to the creation of a corresponding ResponseItem of type [RejectResponseItem]({% link _docs_integrate/data-model-overview.md %}#rejectresponseitem). This will be contained within the `items` property of the [Response]({% link _docs_integrate/data-model-overview.md %}#response) to the Request for proposing Attributes.
+
+### Example of accepting a RequestItemGroup
+
+Let's look at an example where the Sender proposes the Recipient's [PersonName]({% link _docs_integrate/attribute-values.md %}#personname) and contact information in the form of an [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) and a [PhoneNumber]({% link _docs_integrate/attribute-values.md %}#phonenumber) to the Recipient during its onboarding process. For this purpose, the Sender creates a [Request]({% link _docs_integrate/data-model-overview.md %}#request) for proposing Attributes, which contains a ProposeAttributeRequestItem belonging to the PersonName and a RequestItemGroup belonging to the contact information in its `items` property. The [RequestItemGroup]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup) itself includes two ProposeAttributeRequestItems in its `items` property, namely one for the EMailAddress and one for the PhoneNumber. Please note that the `<...>` notation is used as a placeholder for the actual data as usual.
+
+```jsonc
+{
+  "@type": "Request",
+  "items": [
+    {
+      "@type": "ProposeAttributeRequestItem",
+      "mustBeAccepted": true,
+      "attribute": {
+        "@type": "IdentityAttribute",
+        "owner": "",
+        "value": {
+          "@type": "PersonName",
+          "givenName": "<given name that the Sender proposes to the Recipient>",
```

Comment:
Dismiss the "" for the values of givenName and surname, as you don't insert strings here, but attributes.

Reply:
It's possible to use strings here, it's not necessary to use a data object of type GivenName. Since the type of the `givenName` property of the PersonName is specified as `string` on the [Attribute Values page](https://enmeshed.eu/integrate/attribute-values#personname) and not as GivenName, I'd leave it as it is for the time being.

Reply:
object keys are written without "", unless it is necessary (e.g. for "@type")

Reply:
nvm in ts you use keys without "", in json you use ""